### PR TITLE
[AI] Prettify logs

### PR DESCRIPTION
**Pull request by Cjkjvfnby**
- capitalize log entries
- remove duplicated, unclear information
- change formatting

### CODE REVIEW

File: default/AI/AIFleetMission.py [hunk 1]
```diff
@@ -293,31 +293,27 @@ def issue_fleet_orders(self):
...
             self.check_mergers(context=str(fleet_order))
-            if fleet_order.can_issue_order(verbose=True):
+            if fleet_order.can_issue_order(verbose=False):
                 if isinstance(fleet_order, OrderMove) and order_completed:  # only move if all other orders completed
-                    print "\t\t| issuing fleet order %s" % fleet_order
+                    print "Issuing fleet order %s" % fleet_order
                     fleet_order.issue_order()
                 elif not isinstance(fleet_order, OrderMove):
-                    print "\t\t| issuing fleet order %s" % fleet_order
+                    print "Iissuing fleet order %s" % fleet_order
```
Comment (Morlic-fo) [inline default/AI/AIFleetMission.py]: Typo.

File: default/AI/AIFleetMission.py [hunk 2]
```diff
@@ -514,13 +510,13 @@ def __eq__(self, other):
...
             fleet_id = self.fleet.id
-            fleet = self.fleet.get_object()
             return "%-20s [%10s mission]: %3d ships, total rating: %7d target: %s" % (fleet,
                                                                                       AIFleetMissionType.name(self.type),
                                                                                       (fleet and len(fleet.shipIDs)) or 0,
                                                                                       foAI.foAIstate.get_rating(fleet_id).get('overall', 0),
                                                                                       self.target)
         else:
-            return 'Mission with out mission types'
+            return 'Mission of %s with out mission types' % fleet
```
Comment (Morlic-fo) [inline default/AI/AIFleetMission.py]: "without"

File: default/AI/FleetUtilsAI.py [hunk 3]
```diff
@@ -460,11 +460,11 @@ def generate_fleet_orders_for_fleet_missions():
 
     exploration_fleet_missions = foAI.foAIstate.get_fleet_missions_with_any_mission_types([AIFleetMissionType.FLEET_MISSION_EXPLORATION])
     if exploration_fleet_missions:
-        print "Exploration targets: "
+        print "Exploration targets:"
+        for explorationAIFleetMission in exploration_fleet_missions:
+            print " - %s" % explorationAIFleetMission
```
Comment (Morlic-fo) [inline default/AI/FleetUtilsAI.py:465]: variable naming style?

**Comment by Cjkjvfnby** [comment]
> variable naming style?

I don`t declare it in my change set so I did not fix it.  It is theme to other PR

Fixed and rebased.

**Comment by geoffthemedio** [comment]
Any further comments / concerns...?

**Pull request merged**